deploy/deploy-centralvenv.sh

```diff
@@ -0,0 +1,309 @@
+#!/bin/bash
+
+### Usage: Usage: deploy-centralvenv.sh -t <wmcore_tag> -c <central_services_url> [-v wmcore_path] [-p  <patches>] [-s  <service_names>] [-l  <component_list>]
+### Usage:
+### Usage:   -t  <wmcore_tag>       WMCore tag to be used for this deployment [Default: 2.0.0]
+### Usage:   -c  <central_services> Url to central services (e.g. tivanov-unit01.cern.ch)
+### Usage:   -v  <wmcore_path>      WMCore virtual environment target path to be used for this deployment [Default: ./WMCore.venv3]
+### Usage:   -p  <patches>          List of PR numbers
+### Usage:                          (in double quotes and space separated e.g. "5906 5934 5922")
+### Usage:   -s  <service_names>    List of service names to be patched
```

Comment:
it is a legacy, coming from the time (actually still the case) when because of the RPM based deployment every service was having its own area and branch in the deployed WMCore tree. So when it comes to patching a live service it is every time a single patch needs to be applied on all the deployed components (or at least the set of components you know the patch is affecting). Hence the list of services to be patched. And here is a script which I personally use very often when patching a single component [1]. Of course we are talking here about getting rid of slowly getting rid of this structure and pushing it all to run from a single code base and then the list of services to be patched is going to be a redundant option for sure. I do like the idea of running and patching a single code base for both development and runtime. (Although just as a note here - we may have issues if we want multiple users to be running the different services (as it was with cmsweb before), unless we decide to solve that in the K8 deployment scripts - we still have the option to dedicate a set of pods per service so the privileges and authentication related issues may be addressed there). 

[1]
https://github.com/dmwm/WMCore/blob/master/bin/patchComponent.sh

Reply:
Just a note here @vkuznet . 
I did remove the list of services to be patched as you suggested and left only the list of patches to be applied, which is indeed much cleaner, but I still need to implement the actual call to the patching script. Yet to be done.